warehouse/models/staging/rt/stg_rt__feed_fetch_errors.sql

```diff
@@ -0,0 +1,85 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key='event_id',
+        partitions=['current_timestamp()'],
+        cluster_by='group_id',
+    )
+}}
+
+WITH source AS (
+    SELECT * FROM {{ source('sentry_external_tables', 'events') }}
+    WHERE dt IS NOT NULL
+    AND execution_ts = (
+        SELECT MAX(execution_ts) FROM {{ source('sentry_external_tables', 'events') }}
+        WHERE dt IS NOT NULL
+        AND execution_ts IS NOT NULL
+        AND project_slug IS NOT NULL)
```

Comment:
This needs to be templated within `is_incremental()` or a full refresh will not work

Reply:
Oh actually do you mind putting the incremental materialization on the fact table and leaving the staging as a view? That way if we load all events rather than just the fetch exceptions, we will only materialize the exceptions we care about.

Reply:
Resolved by [5d33338](https://github.com/cal-itp/data-infra/pull/2345/commits/5d333383e4f601bb485fb696e636f5131bcc7fc3)